test/assert.js

```diff
@@ -2251,26 +2251,33 @@ describe('assert', function () {
         heroes = ['spiderman', 'superman'],
         fn     = function() { obj.value += 5 },
         fnDec  = function() { obj.value -= 20 },
+        getterFn = function() { return obj.value },
+        getter= function() {},
         bangFn = function() { obj.str += '!' },
         smFn   = function() { 'foo' + 'bar' },
         batFn  = function() { heroes.push('batman') },
         lenFn  = function() { return heroes.length };
 
+    getter.value= obj.value;
     assert.changes(fn, obj, 'value');
+    assert.changes(fn, getterFn, 'value');
```

Comment:
This does not seem correct.

If we want to test properties on functions we must create an `fn` that changes the value of a prop in the `getterFn`. What you've done here is create another `getter` function which has a `value` prop that is never used.